docs/async-await.rst

```diff
@@ -39,6 +39,21 @@ most use cases, but Flask's async support enables writing and using
 code that wasn't possible natively before.
 
 
+Background tasks
+----------------
+
+Async functions will run in an event loop until they complete, at
+which stage the event loop will stop. This means any additional
+spawned tasks that haven't completed when the async function completes
+will be cancelled. Therefore you cannot spawn background tasks, for
```

Comment:
Will they be cancelled or just stopped?

Reply:
ah yes they are cancelled https://github.com/django/asgiref/blob/main/asgiref/sync.py#L240